google_guest_agent/agentcrypto/mtls_mds.go

```diff
@@ -208,3 +207,16 @@ func (j *CredsJob) ShouldEnable(ctx context.Context) bool {
 	_, err := j.client.GetKey(ctx, clientCertsKey, nil)
 	return err == nil
 }
+
+func writeToFile(content []byte, outputFile string) error {
```

Comment:
we might have a few duplicates functions like this in the code base.

Reply:
moved this to utils.